assets/components/service-ca/deployment.yaml

```diff
@@ -23,7 +23,10 @@ spec:
         app: service-ca
         service-ca: "true"
     spec:
-      securityContext: {}
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 1001
```

Comment:
Is there any reason to register or document this user ID to ensure that other workloads do not reuse it?

Reply:
OK, I don't know how UIDs like this are applied. If we have another pod with the same settings, would it be able to see or communicate with this one in a way we wouldn't want?

Reply:
This isn't right. I've override the existing commit. The Cluster Policy Controller should generate the UID based on the security context policy, in this case, `runAsNonRoot`. 

After fixing the cluster role binding, now it works:

```
    securityContext:
      capabilities:
        drop:
        - KILL
        - MKNOD
        - SETGID
        - SETUID
      runAsNonRoot: true
      runAsUser: 1000070000
      ```